### PR TITLE
(DOCSP-970): Updated 2.2 Tableau connectivity.

### PR DESCRIPTION
**Pull request by atsansone**
@steveren : I updated the Tableau section with all of the appropriate information. It is [staged](https://docs-mongodbcom-staging.corp.mongodb.com/bi-connector/anthonysansone/DOCSP-970/tutorial/connecting.html#connect-with-tableau).

### CODE REVIEW

File: source/includes/steps-connect-tableau-mongosql-plugin.yaml [hunk 1]
```diff
@@ -1,80 +1,163 @@
...
+         If this directory does not exist, create it:
+
+         .. class:: copyable-code
+         .. code-block:: sh
+
+            sudo mkdir -p /usr/local/mysql/lib/plugin -m 755
+            sudo chown -R {user}:admin /usr/local/mysql
+            tar -xvf /Users/{user}/Downloads/mongosql-auth-osx-x86_64-v{version}.tgz
+            cp /Users/{user}/Downloads/mongosql-auth-osx-x86_64/lib/mongosql_auth.so ./
+
```
Comment (steveren) [inline source/includes/steps-connect-tableau-mongosql-plugin.yaml]: I would argue that this is too much detail. I know the point here is to provide detailed instructions for Mac users, but I think the commands here are too brittle. It's a judgement call, but I think we have to give the users some credit for being able to follow high-level instructions, like "If this directory does not exist, create it." I think we could leave it at that, along with an instruction to unzip the mongosql_auth archive. On Mac you can double-click the file or use tar -xvf, but I don't think we need to specify.

File: source/includes/steps-connect-tableau-mongosql-plugin.yaml [hunk 2]
```diff
@@ -1,80 +1,163 @@
...
+        * - Platform
+          - TDC File Contents
+
+        * - macOS
+          - 
+
+            .. class:: copyable-code
+            .. literalinclude:: /includes/tdc/mongodb-mac.tdc
+               :language: xml
+               :emphasize-lines: 2-4,7-8
```
Comment (steveren) [inline source/includes/steps-connect-tableau-mongosql-plugin.yaml]: I don't see the value in highlighting certain lines within the .tdc file. They're all equally necessary, even though some are just generic XML tags.

File: source/includes/steps-connect-tableau-mongosql-plugin.yaml (hunk 1 above)
Comment (atsansone) [inline source/includes/steps-connect-tableau-mongosql-plugin.yaml]: This is something we do with Cloud docs quite a bit: being explicit. I added an example frame around it to make clear this is suggested or possible, but not verbatim.

**Comment by breinero** [comment]
LGTM

**Pull request merged**